ensysmod/crud/energy_transmission_distance.py

```diff
@@ -1,30 +1,68 @@
-from typing import List
+from typing import List, Optional
```

Comment:
the more pythonic way is now to go for `value: int | list[int]` for example.
Making the list and optional obsolete.
This is available from 3.10 on, but we might like to change this project wide some time

Reply:
Understood, I was also thinking of making the whole codebase more uniform by adding black and isort as dev dependencies.